corehq/apps/app_manager/migrations/0009_add_sqlglobalappconfig.py

```diff
@@ -37,7 +30,4 @@ class Migration(migrations.Migration):
             name='sqlglobalappconfig',
             unique_together=set([('domain', 'app_id')]),
         ),
-        migrations.RunPython(_populate_sql_global_app_config,
-                             reverse_code=migrations.RunPython.noop,
-                             elidable=True),
```

Comment:
I'm curious why you were able to remove this? It is not clear why the migration would not still need to check if docs exist in couch and migrate them if they do.

Reply:
It does still need to; the check happens in the next migration: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/migrations/0010_sqlglobalappconfig.py#L37